packages/shared/isValidElementType.js

```diff
@@ -28,28 +28,37 @@ import {
 } from 'shared/ReactSymbols';
 
 export default function isValidElementType(type: mixed) {
-  return (
-    typeof type === 'string' ||
-    typeof type === 'function' ||
-    // Note: its typeof might be other than 'symbol' or 'number' if it's a polyfill.
+  if (typeof type === 'string' || typeof type === 'function') return true;
```

Comment:
Nit: please use multiline `if` statements because otherwise it's very easy to miss the early `return`.

Reply:
@gaearon Thanks for code review, I updated it.

Reply:
I think Dan was suggesting:

```js
if (typeof type === 'string' || typeof type === 'function') {
  return true;
}
```

Rather than:
```js
if (typeof type === 'string') return true;
if (typeof type === 'function') return true;
```

Reply:
@bvaughn Thanks, I fixed it